### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by tmccombs**

### CODE REVIEW

File: rules/hocon.bzl [hunk 1]
```diff
@@ -0,0 +1,89 @@
+"""Lucid Hocon compilation rules
+
+Bazel rules for compiling multiple HOCON source files into a flattend HOCON artifact.
+
+In addition to checking syntax and flattening the configuration files, it also verifies that any references to
+lucidBag refer to keys that are available for the applicable IAM roles
+
+[HOCON](https://github.com/lightbend/config)
+
+"""
```
Comment (JaredNeil) [inline rules/hocon.bzl]: Is `lucidBag` also open source? Seems strange to reference it here, if not.

File: WORKSPACE [hunk 2]
```diff
@@ -0,0 +1,64 @@
+workspace(name = "com_lucidchart_rules_hocon")
```
Comment (JaredNeil) [inline WORKSPACE:1]: I would recommend running `buildifier` to keep `BUILD.bazel` and `WORKSPACE` file formatting consistent.
https://github.com/bazelbuild/buildtools/tree/master/buildifier

File: rules/hocon.bzl (hunk 1 above)
Comment (tmccombs) [reply rules/hocon.bzl]: good catch

File: WORKSPACE (hunk 2 above)
Comment (tmccombs) [reply WORKSPACE:1]: done

File: workspace.bzl [hunk 3]
```diff
@@ -0,0 +1,14 @@
+def hocon_repositories():
+    native.maven_jar(
```
Comment (tmccombs) [inline workspace.bzl:2]: @JaredNeil should I do something to add a prefix for this?

**Pull request merged**